ecosystem/models/execution_summary.py

```diff
@@ -27,6 +27,14 @@ def get_warning_logs(self) -> List[str]:
         """Return warning messages."""
         return [log for log in self.logs if "warn" in log.lower()]
 
+    def get_qiskit_depreciation_logs(self) -> List[str]:
+        """Return qiskit depreciation messages."""
+        return [
```

Comment:
Duplicate code of 
https://github.com/qiskit-community/ecosystem/blob/fe9730da03648bdfebaad00dc151f669f2072059/ecosystem/models/execution_summary.py#L49-L53

let's reuse chunks of code in here

Reply:
Indeed